cli.go

```diff
@@ -361,5 +361,8 @@ func (i *CLI) amqpErrorWatcher(amqpConn *amqp.Connection) {
 	if ok {
 		i.logger.WithField("err", err).Error("amqp connection errored, terminating")
 		i.cancel()
+		time.Sleep(2 * time.Minute)
```

Comment:
Is two minutes too long? We only really need to wait for the API calls to GCE/vSphere/etc. to go through to shut down running instances, there's nothing to send back to AMQP, and if the API calls to shut down instances fails we have cleaner scripts that should fix it.